env.cc

```diff
@@ -214,4 +214,13 @@ record *env::getModule(symbol id, string filename)
   return ge.getModule(id, filename);
 }
 
+record *env::getTemplatedModule(symbol id,
+                                string filename,
+                                string index,
```

Comment:
From Andy: 'We call the handle an index later. Is there a difference between the signature handle and the index? If not, we should probably use the same name both places to avoid confusion.'

Reply:
We've set things up to use sigHandle to specifically refer to the handle on the signature. We use Index to refer to an ordered pair (filename, sigHandle). The code has been revised to follow this convention consistently.